orttraining/orttraining/python/training/ortmodule/_inference_manager.py

```diff
@@ -93,21 +98,24 @@ def forward(self, *inputs, **kwargs):
             # If creating the execution agent for the first time, this skip check will not take effect.
             # It will only take effect on subsequent forward calls.
             create_execution_session = False
-            if self._skip_check.is_set(_SkipCheck.SKIP_CHECK_EXECUTION_AGENT) == False or \
-                not self._execution_agent:
+            if self._skip_check.is_set(_SkipCheck.SKIP_CHECK_EXECUTION_AGENT) is False or \
```

Comment:
Should we not continue using `==` and `!=` for values? I think for bools it does not matter. But to keep things consistent, we could probably use `==` and `!=` to denote equality of value and `is` to denote equality of identity?

Reply:
Linter complained. Bools must be compared as `if cond` or `if var is True|False` as per PEP